aws_xray_sdk/ext/sqlalchemy_core/patch.py

```diff
@@ -14,18 +14,21 @@
 def _sql_meta(engine_instance, args):
```

Comment:
Can we add new test case for passwords like `test]password`?

Reply:
The unit test is added.